plugins/pluto/.gitignore

```diff
@@ -0,0 +1,2 @@
+include/plutoplugin/scopy-pluto_export.h
+include/plutoplugin/scopy-pluto_config.h
```

Comment:
this comment was marked as resolved but it still isn't.
gitignore also needs to be modified with the correct path to the plugin include folder